dbms/src/Flash/tests/gtest_aggregation_executor.cpp

```diff
@@ -117,13 +122,12 @@ class ExecutorAggTestRunner : public DB::tests::ExecutorTest
     void executeWithConcurrency(const std::shared_ptr<tipb::DAGRequest> & request, const ColumnsWithTypeAndName & expect_columns)
     {
         WRAP_FOR_DIS_ENABLE_PLANNER_BEGIN
-        for (size_t i = 1; i <= max_concurrency; i += step)
-            ASSERT_COLUMNS_EQ_UR(expect_columns, executeStreams(request, i));
+        ASSERT_COLUMNS_EQ_UR(expect_columns, executeStreams(request));
```

Comment:
Why change this?

Reply:
I feel like it was a waste of resources.
`ASSERT_COLUMNS_EQ_UR(expect_columns, executeStreams(request, 10));` is similar to `ASSERT_COLUMNS_EQ_UR(expect_columns, executeStreams(request, 8));`

Reply:
Maybe we only need to test single-threaded and multi-threaded cases?

Reply:
However, this pr can also not be modified here

Reply:
reverted